### DISCUSSION TITLE
Meeting Agenda: 2023-05-17

### DISCUSSION THREAD

**Discussion by Sephsekla** [WordPress/wp-feature-notifications#319] [Meeting Agendas]
> Please add your suggestions for meeting topics below, and upvote areas you would like to prioritise. I will finalise an agenda on the day prior to the meeting

**Agenda**

- Welcome
- Quick personal updates
- Review of project goals and priorities
- Discussion and help with in-progress issues
- Upcoming events discussions: WCEU and WCUS
- Any other business

**Comment by Sephsekla** [comment]
WordCamp Europe Contributor Day strategy and priorities

**Reply by JasonTheAdams** [reply]
A good standing item would be a review of project goals and priorities.

**Reply by Sephsekla** [reply]
Going to move this into its own comment so it can be specifically voted on

**Comment by Sephsekla** [comment]
> A good standing item would be a review of project goals and priorities.

props @JasonTheAdams

**Reply by johnhooks** [reply]
On a little bit separate note, it would be nice to have a place we keep that written down and updated. Would the meeting notes be sufficient? ... also, are meeting notes kept? If so, where?

**Reply by Sephsekla** [reply]
Right now, not really, certainly nothing substantial. But in tandem with the agenda I'd like to start keeping notes properly and publish them on here.

**Reply by johnhooks** [reply]
To me the review of the purpose and objectives is important to keep updated in the notes, not tracking individual issues.

**Comment by johnhooks** [comment]
Open time to bring up PR/Issues that need help moving forward.
